initdb.d/01_main.sql

```diff
@@ -1,339 +1,143 @@
--- \connect lega
+CREATE SCHEMA sda;
 
--- We use schemas for isolation instead of multiple databases.
--- Look at the grants.sql file, for access user/role rights setup
+SET search_path TO sda;
 
-CREATE SCHEMA local_ega; -- includes the main table, some views and errors
-
-SET search_path TO local_ega;
-
-CREATE TYPE checksum_algorithm AS ENUM ('MD5', 'SHA256', 'SHA384', 'SHA512'); -- md5 is bad. Use sha*!
+-- ENUMS
+CREATE TYPE checksum_algorithm AS ENUM ('MD5', 'SHA256', 'SHA384', 'SHA512');
+CREATE TYPE encryption_algorithm AS ENUM ('CRYPT4GH', 'PGP', 'AES');
 CREATE TYPE storage AS ENUM ('S3', 'POSIX');
```

Comment:
i am not sure why we would keep this around. We configure this in `sda-pipeline` already and we cannot use multiple storage types on a database, additionally i don't see any meaningful queries we can have where this would be useful. So my suggestion is to remove it.

Reply:
I agree, until someone misses it, we can remove storage type.